toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -46,6 +46,7 @@ var (
 	disableDefaultRepos  = app.Flag("disable-default-repos", "Disable pulling packages from PMC repos").Bool()
 	disableUpstreamRepos = app.Flag("disable-upstream-repos", "Disables pulling packages from upstream repos").Bool()
 	toolchainManifest    = app.Flag("toolchain-manifest", "Path to a list of RPMs which are created by the toolchain. Will mark RPMs from this list as prebuilt.").ExistingFile()
+	repoSnapshotTime     = app.Flag("repo-snapshot-time", "Repo time limit for tdnf virtual snapshot").String()
```

Comment:
We should clarify this is optional.  Is it optional?

Reply:
Yes the flag is optional, I will add a note in the description.